js/components/userFeedback/css/styles.css

```diff
@@ -0,0 +1,6 @@
+#message-id a {
+  color: #ffe970;
+}
+#message-id a:visited, #message-id a:hover, #message-id a:active, #message-id a:focus {
+  color: #edd656;
+}
```

Comment:
move CSS to JS `const useStyles = makeStyles(theme => ({
  contColButton: {
    '&:hover': {
      backgroundColor: theme.palette.primary.main,
      color: theme.palette.primary.contrastText
    }}
}))
`